firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -99,6 +99,10 @@ class BaseTestCase: XCTestCase {
         } else {
             app.launchArguments = [LaunchArguments.PerformanceTest] + launchArguments
         }
+
+        // FXIOS-13129: Remove these arguments once we migrate existing tests to support homepage redesign
```

Comment:
@dragosb01 let's start working on the tests on a branch so that we know the changes needed with the experiment ON, most of those are already in my comment here. Thanks!